No MTF Tesla/Handlers/Player.cs

```diff
@@ -7,22 +7,7 @@ class Player
     {
         public void TriggeringTeslaGate(TriggeringTeslaEventArgs ev)
         {
-            if (Singleton.Config.MTFTesla) return;
-            else
-            {
-                if (ev.Player.Team == Team.MTF)
-                {
-                    ev.IsTriggerable = false;
-                }
-            }
-            if (Singleton.Config.ScientistTesla) return;
-            else
-            {
-                if (ev.Player.Team == Team.RSC)
-                {
-                    ev.IsTriggerable = false;
-                }
-            }
+            ev.IsTriggerable = !((Singleton.Config.MTFTesla && ev.Player.Team == Team.MTF) || (Singleton.Config.ScientistTesla && ev.Player.Team == Team.RSC));
```

Comment:
> if some other plugin has already set IsTriggerable = false, this plugin will then make the tesla turn on.
and the owners be like: " why doesn't my XXXXX plugin work and the tesla still turns on?"
it's easier to debug why the tesla doesn't turn on than to debug "why it turns on".
>
> @ Nekonyx

Reply:
do i pull this or no? i cant tell if this is for my plugin or this PR

Reply:
Oops fixed it.

Reply:
As Pint fixed it, free feel to merge